flang/test/Examples/ctofortran.f90

```diff
@@ -0,0 +1,59 @@
+! UNSUPPORTED: system-windows
```

Comment:
Can we also add something like `REQUIRES: clang`

Reply:
Thanks, Kiran.  Will do.